hedera-node/src/test/java/com/hedera/services/store/models/AccountTest.java

```diff
@@ -304,7 +310,44 @@ public void createAccountFromGrpcTransaction() {
 		assertEquals(created.getAutoRenewSecs(), customAutoRenewPeriod);
 		assertEquals(created.getMaxAutomaticAssociations(), maxAutoAssociations);
 	}
-	
+
+	@Test
+	public void updateAccountFromGrpcTransaction() {
+		var op =
+				CryptoUpdateTransactionBody.newBuilder()
+						.setKey(key)
+						.setMemo(StringValue.of(memo))
+						.setAutoRenewPeriod(Duration.newBuilder().setSeconds(customAutoRenewPeriod))
+						.setExpirationTime(Timestamp.newBuilder().setSeconds(1_000_000).build())
+						.setProxyAccountID(proxy)
+						.setReceiverSigRequiredWrapper(BoolValue.of(true))
+						.setMaxAutomaticTokenAssociations(Int32Value.of(maxAutoAssociations))
+						.build();
+
+		subject.updateFromGrpc(
+				Optional.of(op.getKey()),
+				Optional.of(op.getMemo()),
+				Optional.of(op.getAutoRenewPeriod()),
+				Optional.of(op.getExpirationTime()),
+				Optional.of(op.getProxyAccountID()),
+				Optional.of(op.getReceiverSigRequiredWrapper()),
+				Optional.of(op.getMaxAutomaticTokenAssociations())
+		);
```

Comment:
I don't see the need to build a whole gRPC transaction in order to get the separate elements. Can we use them directly?

Reply:
Agree, will use them directly.